packages/ui-library/src/components/form-caption/index.ts

```diff
@@ -16,39 +16,40 @@ export class BlrFormCaption extends LitElement {
   @property() message?: string;
   @property() icon?: SizelessIconType;
   @property() variant: CaptionVariantType = 'hint';
-  @property() size?: FormSizesType = 'md';
+  @property() sizeVariant?: FormSizesType = 'md';
   @property() childElement?: TemplateResult<1>;
   @property() theme: ThemeType = 'Light';
+  @property({ type: Boolean }) hasIcon?: boolean;
 
   protected render() {
-    if (this.size) {
+    if (this.sizeVariant) {
       const dynamicStyles = this.theme === 'Light' ? [captionLight] : [captionDark];
 
       const classes = classMap({
         'blr-form-caption': true,
         [`${this.variant}`]: this.variant,
-        [`${this.size}`]: this.size,
+        [`${this.sizeVariant}`]: this.sizeVariant,
       });
 
       const iconClasses = classMap({
         'blr-icon': true,
-        [this.size]: this.size,
+        [this.sizeVariant]: this.sizeVariant,
       });
 
       const iconSizeVariant = getComponentConfigToken([
         'cmp',
         'CaptionComponent',
         'Icon',
         'SizeVariant',
-        this.size.toUpperCase(),
+        this.sizeVariant.toUpperCase(),
       ]).toLowerCase() as SizesType;
 
       return html`
         <style>
           ${dynamicStyles}
         </style>
         <div class=${classes}>
-          ${this.icon
+          ${this.hasIcon
```

Comment:
I think this is wrong. The icon prop should still be there.

But what is missing that we want to newly introduce the "hasIcon" property in a similar way how we have it in the Input Field Text (where it is still named showInputIcon"). If this new property is false, there should not be any icon in the component example story and the icon prop should also be hidden in the prop panel.

Would be cool if you can add this.

Reply:
I have changed it to showIcon but it just means that we have changed the name from hasIcon to ShowIcon.

Reply:
Looking at the newest version in Storybook, this seems to work correct now. But the property name should still be "hasIcon" instead of "showIcon". Can you change that?

Another small issue I found is, that the "message" prop should also be within the "Content / Settings" section in the prop panel. Currently it is placed in its own section called "Content / Setting". It should be placed before the "hasIcon" prop.